vehicle:
  wheel_base          : 2   # distance meter
  dist_front_rear_wheels : 1 # distance meter

patrol:
  path_topic           : "odom_path"
#  odom_topic           : "/mavros/local_position/odom"
  wait_time_at_ends   : 5  # secs
  google_map_center_always : true
  min_speed: 0.3
  max_speed: 3

pure_pursuit:
  min_look_ahead : 3 # meters
  max_look_ahead : 6 # meters
  speed_to_lookahead_ratio : 2
  speed_curvature_ratio : 2
  

stanley:
  control_gain : 0.1 # meters
  softening_gain : 1 # meters